cmd/kubectl-gadget/trace/trace.go

```diff
@@ -21,19 +21,12 @@ import (
 
 	commonutils "github.com/kinvolk/inspektor-gadget/cmd/common/utils"
 	"github.com/kinvolk/inspektor-gadget/cmd/kubectl-gadget/utils"
-	eventtypes "github.com/kinvolk/inspektor-gadget/pkg/types"
 
 	"github.com/spf13/cobra"
 )
 
 type TraceEvent interface {
 	any
-
-	// The Go compiler does not support accessing a struct field x.f where x is
-	// of type parameter type even if all types in the type parameter's type set
-	// have a field f. We may remove this restriction in Go 1.19. See
-	// https://tip.golang.org/doc/go1.18#generics.
```

Comment:
What about adding this comment on `BaseElement`?

Reply:
Agree